python/nav/web/ipdevinfo/views.py

```diff
@@ -334,6 +334,11 @@ def get_cam_info(mac):
             }
             sensor_metrics.append(metric)
         find_rules(sensor_metrics)
+
+        netbox.data = {
+            key: value for key, value in netbox.data.items() if not key.startswith('__')
+        }
+
```

Comment:
I'd rather not do it this way: This essentially modifies the `Netbox` object itself, and if this object happens to be saved back to the database, you have just effectively deleted the data.

I'd look at how the template iterates the data attribute, and leave out the double-underscore-ones there.